tests/main.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Probably shouldn't merge this empty

Reply:
Aah forgot about this. I think I should prolly get rid of it for now. I really want to add integration tests, but I have  a couple of things I wanted to add first.

Reply:
I don't really care about it being full or not. It's just that if it's empty, it's just clutter. Better to delete now than to forget.

There should really be an "empty file" clippy lint...